packages/core/src/components/Toggle/Toggle.stories.mdx

```diff
@@ -1,15 +1,25 @@
 import { Toggle } from './Toggle';
-import { Preview, Story, Meta, Props } from '@storybook/addon-docs/blocks';
+import { Preview, Story, Meta, Props} from '@storybook/addon-docs/blocks';
 import * as stories from './Toggle.stories';
+import { boolean, select } from '@storybook/addon-knobs';
+import { size, labelPosition, variants, colors, ThemeInterface } from './Toggle.stories';
```

Comment:
There are no `variants, colors` exported from `Toggle.stories`. Also we should change name of `size` &  `labelPosition ` to `sizes` &  `labelPositions `